### PR TITLE
BZ1969693 - 4.8 LSO change to merge Daemonsets

### PR DESCRIPTION
**Pull request by lpettyjo**
For version 4.8+

https://bugzilla.redhat.com/show_bug.cgi?id=1969693

Direct link to preview: https://deploy-preview-33638--osdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-volume-cr_persistent-storage-local

See https://github.com/openshift/openshift-docs/pull/31742 for changes to rel notes for this issue.

PTAL @chao007, @jsafrane

### CODE REVIEW

File: modules/persistent-storage-local-create-cr.adoc [hunk 1]
```diff
@@ -52,7 +52,7 @@ spec:
 <4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
 <5> The file system that is created when the local volume is mounted for the first time.
 <6> The path containing a list of local storage devices to choose from.
-<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.
+<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/disk/by-id/xxx`. PVs are created for these local disks when the provisioner is deployed successfully.
 +
```
Comment (gnufied) [inline modules/persistent-storage-local-create-cr.adoc:56]: nit: lets use `wwn`  as placeholder for these disk names. They make bit more sense as disk ids - https://en.wikipedia.org/wiki/World_Wide_Name

**Comment by ahardin-rh** [comment]
Looks good!

**Pull request merged**